examples/hdr/rs-hdr.cpp

```diff
@@ -4,15 +4,20 @@
 #include <librealsense2/rs.hpp> // Include RealSense Cross Platform API
 #include "example.hpp"          // Include short list of convenience functions for rendering
 #include <iostream>
+#include "imgui.h"
+#include "imgui_impl_glfw.h"
 
-// HDR Example demonstrates how to
-// use the HDR feature - only for D400 product line devices
-int main(int argc, char * argv[]) try
+enum frame_id { IR1, IR2, DEPTH1, DEPTH2, HDR };
+#define EXPOSURE_START_VALUE_HIGH 6000
```

Comment:
We try to avoid macros whenever possible -  replace it with const(expr) uint ...